libs/core/algorithms/include/hpx/parallel/container_algorithms/partial_sort_copy.hpp

```diff
@@ -307,14 +312,15 @@ namespace hpx { namespace ranges {
     ///           range_iterator_t<Rng2>> otherwise.
     ///           The algorithm returns {last, result_first + N}.
     ///
-    template <typename ExPolicy, typename FwdIter, typename Sent1,
-        typename RandIter, typename Sent2, typename Comp, typename Proj1,
-        typename Proj2>
+    template <typename ExPolicy, typename Rng1, typename Rng2,
+        typename Compare = ranges::less,
+        typename Proj1 = parallel::util::projection_identity,
+        typename Proj2 = parallel::util::projection_identity>
     parallel::util::detail::algorithm_result_t<ExPolicy,
         partial_sort_copy_result<hpx::traits::range_iterator_t<Rng1>,
         hpx::traits::range_iterator_t<Rng2>>>
     partial_sort_copy(
-        ExPolicy&& policy, Rng1&& rng1, Rng2&& rng2, Comp&& comp = Comp(),
+        ExPolicy&& policy, Rng1&& rng1, Rng2&& rng2, Compare&& comp = Compare(),
```

Comment:
Why do we use `Compare` here instead of `Comp`? I haven't seen it again so far

Reply:
It's just a name, but consistency matters. Please feel free to change it.

Reply:
So, I will change `Compare&& comp = Compare()` to `Comp&& comp = Comp()` also in the code part. Is that ok?